druntime/src/core/sys/posix/sys/socket.d

```diff
@@ -556,6 +556,7 @@ version (linux)
             SO_RCVLOWAT     = 18,
             SO_RCVTIMEO     = 20,
             SO_REUSEADDR    = 2,
+            SO_REUSEPORT    = 15,
```

Comment:
this appears to be unrelated to `LoongArch64`

Reply:
Thanks for the review!

I am not an expert in this... Could you let me know why is it so (and the one below)?  I simply copied this definitions from the kernel headers with:

```gcc -E -P -include sys/socket.h - <<< "SO_REUSEPORT"```

Reply:
If it's not in the posix standard, then it won't be included here, rather in the Linux specific system socket module.

Reply:
Oh, I see. Thanks.